packages/nuxi/src/commands/dev.ts

```diff
@@ -96,7 +97,8 @@ export default defineNuxtCommand({
           dev: true,
           ready: false,
           overrides: {
-            logLevel: args['log-level']
+            logLevel: args['log-level'],
+            ...(options.config || {})
```

Comment:
`config` looks good to me but also wondering if `overrides` would give more consistency for options naming (and also more clarification this is strictly overriding any level of config)

```suggestion
            ...(options.overrides || {})
```